app/http.php

```diff
@@ -0,0 +1,1039 @@
+                throw new Exception('Failed to create build container', 500);
+            }
+
+            $orchestration->networkConnect($runtimeId, App::getEnv('OPEN_RUNTIMES_EXECUTOR_NETWORK', 'executor_runtimes'));
+
+            /**
+             * Execute any commands if they were provided
+             */
+            if (!empty($commands)) {
+                $status = $orchestration->execute(
+                    name: $runtimeId,
+                    command: $commands,
+                    stdout: $stdout,
+                    stderr: $stderr,
+                    timeout: App::getEnv('OPEN_RUNTIMES_EXECUTOR_BUILD_TIMEOUT', 900)
+                );
+
+                if (!$status) {
+                    throw new Exception('Failed to build dependenices ' . $stderr, 500);
+                }
+            }
+
+            /**
+             * Move built code to expected build directory
+             */
+            if (!empty($destination)) {
+                // Check if the build was successful by checking if file exists
+                if (!\file_exists($tmpBuild)) {
+                    throw new Exception('Something went wrong during the build process', 500);
+                }
+
+                $destinationDevice = getStorageDevice($destination);
+                $outputPath = $destinationDevice->getPath(\uniqid() . '.' . \pathinfo('code.tar.gz', PATHINFO_EXTENSION));
+
+                $buffer = $localDevice->read($tmpBuild);
+                if (!$destinationDevice->write($outputPath, $buffer, $localDevice->getFileMimeType($tmpBuild))) {
+                    throw new Exception('Failed to move built code to storage', 500);
+                };
+
+                $container['outputPath'] = $outputPath;
+            }
+
+            /** @phpstan-ignore-next-line */
+            if (empty($stdout)) {
+                $stdout = 'Build Successful!';
+            }
+
+            $endTimeUnix = \microtime(true);
+            $duration = $endTimeUnix - $startTimeUnix;
+
+            $container = array_merge($container, [
+                'status' => 'ready',
+                'stdout' => \mb_strcut($stdout, 0, 1000000), // Limit to 1MB
+                'stderr' => \mb_strcut($stderr, 0, 1000000), // Limit to 1MB
+                'startTimeUnix' => (int) $startTimeUnix,
+                'endTimeUnix' => (int) $endTimeUnix,
+                'duration' => $duration,
+            ]);
+
+            if (!$remove) {
+                $activeRuntimes->set($activeRuntimeId, [
+                    'id' => $containerId,
+                    'name' => $activeRuntimeId,
+                    'hostname' => $runtimeHostname,
+                    'created' => (int) $startTimeUnix,
+                    'updated' => \time(),
+                    'status' => 'Up ' . \round($duration, 2) . 's',
+                    'key' => $secret,
+                ]);
+            }
+
+            Console::success('Build Stage completed in ' . ($duration) . ' seconds');
```

Comment:
Let's remove all the logs from the HTTP server, they will become noisy very soon.

Reply:
All removed.